src/main/java/seedu/address/logic/parser/UntagEventCommandParser.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.UntagEventCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new UntagEventCommand object
+ */
+public class UntagEventCommandParser implements Parser<UntagEventCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the TagPersonsToEventCommand
+     * and returns an TagPersonsToEventCommand object for execution.
```

Comment:
Should be `UntagEventCommand`

```suggestion
     * Parses the given {@code String} of arguments in the context of the UntagEventCommand
     * and returns an UntagEventCommand object for execution.
```